### DISCUSSION TITLE
Lidl LIVARNO LUX LED Light Bar & Mood Light

### DISCUSSION THREAD

**Discussion by FlexxFR** [Koenkk/zigbee2mqtt#5944] [Devices]
Hi All,

just connected both the new Lidl smart home products,  Light Bar & Mood light. (Both zigbee 3.0 according the manual)
They work great with Zigbee2Mqtt and paired right away when switched on for the first time!

As with other Lidl products they are labeled TuYa products. Strange thing is that both the Light Bar as the Mood Light are recognized as TuYa TS0505A.
https://www.zigbee2mqtt.io/devices/TS0505A.html#tuya-ts0505a

From the logs:
_Device '0x588e81fffefec8a7' is supported, identified as: TuYa RGB+CCT light controller (TS0505A)__

This is only a small annoyance, I don't know if this can be solved and how? Is there anything specific in the log, during pairing that can distinguish this device from a normal TS0505A, or is this device exactly the same only in a different form-factor?

Anyway, not a big deal.

**Comment by Koenkk** [comment]
Please provide the data/database entries for these device + actual model numbers (probably start with HG*) and a link to the Lidl site with picture.

**Comment by Fridolin34** [comment]
Don't know, whether this is the right entry, but I would just like to help.

Database entry in my system is:
{"id":19,"type":"Router","ieeeAddr":"0x588e81fffefb9cf2","nwkAddr":34470,"manufId":4098,"manufName":"_TZ3000_gek6snaj","powerSource":"Mains (single phase)","modelId":"TS0505A","epList":[1,242],"endpoints":{"1":{"profId":260,"epId":1,"devId":269,"inClusterList":[0,3,4,5,6,8,768,4096],"outClusterList":[25,10],"clusters":{"genBasic":{"attributes":{"modelId":"TS0505A","manufacturerName":"_TZ3000_gek6snaj","powerSource":1,"zclVersion":3,"appVersion":80,"stackVersion":0,"hwVersion":1,"dateCode":""}},"genLevelCtrl":{"attributes":{"currentLevel":254}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":80,"stackVersion":0,"hwVersion":1,"dateCode":"","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1611853367696}

Lidl website:
https://www.lidl.de/de/livarno-lux-lichtleiste-mit-lichtfarbensteuerung/p365241

But I cannot find anything concerning a model number starting with HG... The manual just labels it with the model number 14149505L and the IAN 341729_2001.

**Reply by Koenkk** [reply]
Thanks added, looks `14149505L` is  indeed the model number. 

Both mood light and light bar will be explicitly supported in the next release (on 1 feb)

**Reply by Fridolin34** [reply]
Thanks a lot.

**Reply by FlexxFR** [reply]
noob question but where can I find the data/database info?
Is it still required?

The model number on my box is slightly different! 
It's 1414950**6**L for the Smart LED Light Bar. (White model, there is also a black model)
https://www.lidl.nl/nl/p/product-recommendation/rgb-led-sfeerlamp/p44955

The model number for the Smart Led Mood Light is 14148906L
https://www.lidl.nl/nl/p/smart-home/rgb-led-sfeerverlichting/p44954

If I need to look something from the data/database, please give me a hint how to find that :)

**Reply by Koenkk** [reply]
@Fridolin34 do you have a black one

@FlexxFR you can find it in the zigbee2mqtt install directory in the `data` folder (`database.db` can be opened with a text editor)

**Reply by Fridolin34** [reply]
Yes, mine is black.

**Reply by FlexxFR** [reply]
Ok, found it:

Smart LED Light Bar. (White model) model number 14149506L:
https://www.lidl.nl/nl/p/product-recommendation/rgb-led-sfeerlamp/p44955

{"id":45,"type":"Router","ieeeAddr":"0x588e81fffede9683","nwkAddr":2903,"manufId":4098,"manufName":"_TZ3000_gek6snaj","powerSource":"Mains (single phase)","modelId":"TS0505A","epList":[1,242],"endpoints":{"1":{"profId":260,"epId":1,"devId":269,"inClusterList":[0,3,4,5,6,8,768,4096],"outClusterList":[25,10],"clusters":{"genBasic":{"attributes":{"modelId":"TS0505A","manufacturerName":"_TZ3000_gek6snaj","powerSource":1,"zclVersion":3,"appVersion":80,"stackVersion":0,"hwVersion":1,"dateCode":""}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":80,"stackVersion":0,"hwVersion":1,"dateCode":"","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1611864720647}


The Smart Led Mood Light with model number 14148906L:.
https://www.lidl.nl/nl/p/smart-home/rgb-led-sfeerverlichting/p44954

{"id":44,"type":"Router","ieeeAddr":"0x842e14fffe05e102","nwkAddr":37024,"manufId":4098,"manufName":"_TZ3000_9cpuaca6","powerSource":"Mains (single phase)","modelId":"TS0505A","epList":[1,242],"endpoints":{"1":{"profId":260,"epId":1,"devId":269,"inClusterList":[0,3,4,5,6,8,768,4096],"outClusterList":[25,10],"clusters":{"genBasic":{"attributes":{"modelId":"TS0505A","manufacturerName":"_TZ3000_9cpuaca6","powerSource":1,"zclVersion":3,"appVersion":80,"stackVersion":0,"hwVersion":1,"dateCode":""}},"lightingColorCtrl":{"attributes":{"colorTemperature":150,"currentX":22378,"currentY":11189}},"genOnOff":{"attributes":{"onOff":0}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":80,"stackVersion":0,"hwVersion":1,"dateCode":"","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1611817973417}

**Reply by Koenkk** [reply]
Added!

**Reply by l00kanio** [reply]
I will jump on the bandwagon ;-) I have  black 14149505L, IAN 365268_2101, no RGB (only color temp.) AFAIK, which is found as follows: 

{"id":6,"type":"Router","ieeeAddr":"0xb4e3f9fffee16115","nwkAddr":61037,"manufId":4098,"manufName":"_TZ3210_iystcadi","powerSource":"Mains (single phase)","modelId":"TS0505B","epList":[1,242],"endpoints":{"1":{"profId":260,"epId":1,"devId":269,"inClusterList":[0,3,4,5,6,4096,8,768,61184],"outClusterList":[25,10],"clusters":{"genBasic":{"attributes":{"modelId":"TS0505B","manufacturerName":"_TZ3210_iystcadi","powerSource":1,"zclVersion":3,"appVersion":66,"stackVersion":0,"hwVersion":1,"dateCode":""}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":66,"stackVersion":0,"hwVersion":1,"dateCode":"","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1644011519972,"defaultSendRequestWhen":"immediate"}

@Koenkk  I would be thankful for adding it :)  I tried do it myself but apparently I am not skilled enough :(

**Comment by Miruzooo** [comment]
https://www.lidl.sk/p/livarno-home-svetelna-lista-s-led-zigbee-smart-home/p100339565  Black

{"id":27,"type":"Router","ieeeAddr":"0xb4e3f9fffec6b5fb","nwkAddr":13853,"manufId":4098,"manufName":"_TZ3210_iystcadi","powerSource":"Mains (single phase)","modelId":"TS0505B","epList":[1,242],"endpoints":{"1":{"profId":260,"epId":1,"devId":269,"inClusterList":[0,3,4,5,6,4096,8,768,61184],"outClusterList":[25,10],"clusters":{"genBasic":{"attributes":{"modelId":"TS0505B","manufacturerName":"_TZ3210_iystcadi","stackVersion":0,"dateCode":"","zclVersion":3}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":66,"stackVersion":0,"hwVersion":1,"dateCode":"","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1650997904445,"defaultSendRequestWhen":"immediate"}

Please add

THX

**Reply by Koenkk** [reply]
`_TZ3210_iystcadi`  will be supported in the next release on 1 May

**Reply by mtrr** [reply]
It works "kinda". RGB only functions after power cycling the device. As soon as I try to change brightness or colour temperature the white LEDs come on and never get switched off, no matter what setting I try.

cheers

**Reply by Lynetk** [reply]
I have the same experience

**Reply by ghost** [reply]
Hello, is there already a solution for the problem?

Thanks.

**Reply by SmartLiving-Rocks** [reply]
Just updated to Version 1.27.0 and I have the same behavior as @mtrr describes.

**Reply by mtrr** [reply]
seems like we need to create an issue, otherwise nobody will take care of our problem. I'm on vacation and away, mobile phone only...

**Reply by SmartLiving-Rocks** [reply]
ok thanks - have a nice holiday!

**Reply by crooy** [reply]
> seems like we need to create an issue, otherwise nobody will take care of our problem. I'm on vacation and away, mobile phone only...

#13421
